pkg/covermerger/bq_csv_reader.go

```diff
@@ -67,7 +67,7 @@ func (r *bqCSVReader) InitNsRecords(ctx context.Context, ns, filePath, commit st
 			AS (
 				SELECT
 					kernel_repo, kernel_branch, kernel_commit, file_path, sl, SUM(hit_count) as hit_count
-				FROM syzkaller.syzbot_coverage.%s
+				FROM syzkaller.syzbot_coverage.`+"`%s`"+`
```

Comment:
I wonder if it had been better to use query parameters functionality offered by the library:
https://pkg.go.dev/cloud.google.com/go/bigquery#hdr-Querying

I'd expect it to do all the necessary escaping automatically.

Reply:
It is a table name. Do you think it could be parametrized?

Reply:
https://cloud.google.com/bigquery/docs/parameterized-queries

"Query parameters can be used as substitutes for arbitrary expressions. Parameters cannot be used as substitutes for identifiers, column names, table names, or other parts of the query."

indeed.

Reply:
It is possible  SELECT FROM syzkaller.syzbot_coverage.* and iterate over the tables.
But it looks too complex for this case.

Reply:
"This feature is only available with [GoogleSQL syntax](https://cloud.google.com/bigquery/docs/reference/standard-sql)."
I use PostgreSQL syntax.

Reply:
@a-nogikh PTAL. If you know the better options to select the table by name please let me know.

Reply:
#5202 to continue this topic